### ISSUE TITLE
Ingestion request EAMT 2022

### ISSUE THREAD

**Issue by BramVanroy** [acl-org/acl-anthology#1883]
Hello

We are preparing the proceedings for the EAMT 2022 conference. We found on [this website](https://aclanthology.org/info/contrib/) that we should start an issue here to get the proceedings in the anthology. 

Repository that we are tracking the proceedings in [this repo](https://github.com/ardate/EAMT2022-proceedings/) (WIP, based on previous edition of EAMT 2020). [Meta data](https://github.com/ardate/EAMT2022-proceedings/blob/master/ACLPUB/info/meta.txt) (correct). So the venue name is `eamt` and there is only one volume (main conference), as title `Proceedings of the 23nd Annual Conference of the European Association for Machine Translation`.

Conference starts June 1st. We would like to negotiate 29th of May as a publication date.

Do you need anything else from us?

**Comment by mjpost** [comment]
Hi,

* That pub date sounds fine!
* [Proceedings title check out](https://aclanthology.org/venues/eamt/)
* In the meta files, can you list chairs in BibTeX format (last name, first name(s))
* Are there any workshops?

**Comment by BramVanroy** [comment]
Do you mean in [this meta file](https://github.com/ardate/EAMT2022-proceedings/blob/master/ACLPUB/info/meta.txt)? We based ourselves on last years edition's [meta](https://github.com/fmmb/EAMT2020-proceedings/blob/master/ACLPUB/info/meta.txt), which follows the same format. But perhaps you are referring to other meta files?

There won't be any workshops, so only the main conference.

**Comment by mjpost** [comment]
Yes, the meta file. We can automatically parse names into “last, first” but it is not perfect. So better to list it correctly.

**Comment by BramVanroy** [comment]
I updated [the file](https://github.com/ardate/EAMT2022-proceedings/blob/master/ACLPUB/info/meta.txt). Does that look good?

**Comment by mjpost** [comment]
Yes (and thanks for linking it!)

**Comment by BramVanroy** [comment]
@mjpost Per the instructions: I've uploaded the processed proceedings [here](https://lt3.ugent.be/media/uploads/eamt2022/eamt-2022_data.tgz). Is there anything else you need from us? Thanks!

**Comment by BramVanroy** [comment]
Note: a late sponsors has been added, so the archive has been updated to reflect that a moment ago.

**Comment by mjpost** [comment]
Hi @BramVanroy—given the date, it seems that you decided to submit later? Just thought I'd check.

**Comment by mjpost** [comment]
Hi @BramVanroy—sorry, we are behind on this, but will get to it right now.

**Comment by BramVanroy** [comment]
Hi @mjpost. We've received an email of an author who accidentally left out one page in their camera-ready version. That means that if we were to updated the proceedings with a new version, all papers that come after their work will now have a different page number. This is not ideal because that would invalidate the current references. So how should we approach this? Is there still time to make such a significant change?

**Comment by mjpost** [comment]
Hi, at this point it needs to be processed as a revision: https://aclanthology.org/info/corrections/
